Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml

```
@@ -12,7 +12,7 @@
   startingGear: ForensicMantisGear
   icon: "JobIconForensicMantis"
   supervisors: job-supervisors-rd
-  antagAdvantage: 5 # DeltaV - From 4 to 5
+  antagAdvantage: 0 # DeltaV - From 4 to 0
```

```suggestion
  #antagAdvantage: # DeltaV - Removes AntagAdvantage
```
As a side note: This should probably be moved into our own folders at some point..

---

Yeah, in another PR